template/2-history.md

```diff
@@ -1,19 +1,15 @@
 # History
 
-*Maximum 1 page*
+Key features of the V1 PoC included Google authentication for a seamless and secure login experience. Users could easily find activities around them, either displayed on an interactive map or in a list format, based on their current location or a selected place. This feature ensured that users could discover new hiking, climbing, biking, and trail running spots effortlessly.
 
-*Describe the V1 POC as planned by Sprint10.*
--Login through Google authetication\
--Find activities around you, either on the map or in a list format based either on user location or a chosen place\
--Save those activities\
--Save user preferences\
--Available in 3 languages\
--Can filter through activities\
--Step counter\
--Save information about the activities an user has done\
--Download Activities to access them while offline\
--Add friends\
--Send messages to friends\
-*What did you learn?*
+To enhance user experience, the app allowed users to save activities and their preferences, making it simple to revisit favorite spots or receive tailored recommendations. Recognizing the diversity of language in Switzerland, Lasta was made available in three languages, ensuring accessibility for a wider audience. Additionally, users could filter through activities to find exactly what they were looking for, whether it was an easy hiking trail or a challenging climbing route.
 
-*What is missing to bridge from PoC to MVP?*
+The V1 PoC also included a step counter, allowing users to track their physical activity and stay motivated. Users could save information about the activities they had completed, providing a personal log of their outdoor adventures. To cater to the needs of users in areas with limited connectivity, Lasta enabled the downloading of activities for offline access.
+
+Social features were also a significant part of the PoC. Users could add friends within the app and send messages, fostering a sense of community and making it easier to plan group activities.
+
+From the V1 PoC, we learned that the core functionalities were well designed: it it easy to find and save activities. The multilingual support is crucial, as it broadens the app's appeal. However, we also identified areas that are needing improvement: more personalized recommendations and advanced filtering options for example.
+
+To transition from the PoC to a Minimum Viable Product, several key enhancements are necessary. First, we need to refine the recommendation engine to provide more personalized and accurate suggestions based on user preferences and activity history. Improving the filtering capabilities will also be crucial, allowing users to narrow down activities more effectively.
```

Comment:
For this part, I would suggest you to add the fact that the app still lacks of activity. We might need to find an other source of data and combine it with OSM to have enough activities in all categories.

Reply:
Maybe also adding some new sports